### PR TITLE
Add dutch translation

### PR DESCRIPTION
**Pull request by ngohungphuc**

### CODE REVIEW

File: Stats/Supporting Files/nl.lproj/Localizable.strings [hunk 1]
```diff
@@ -0,0 +1,212 @@
+//
+//  Localizable.strings
+//  Stats
+//
+//  Created by Phúc Ngô on 02/04/2020.
+//  Running on macOS Big Sur M1.
+//
+//  Copyright © 2020 Phúc Ngô. All rights reserved.
```
Comment (exelban) [inline Stats/Supporting Files/nl.lproj/Localizable.strings:8]: Hi. Thanks for the translation. Could you please change the copyright to me?

Comment (ngohungphuc) [reply Stats/Supporting Files/nl.lproj/Localizable.strings]: Hi @exelban Done.

File: Stats/Supporting Files/nl.lproj/Localizable.strings [hunk 2]
```diff
@@ -5,7 +5,7 @@
 //  Created by Phúc Ngô on 02/04/2020.
```
Comment (exelban) [inline Stats/Supporting Files/nl.lproj/Localizable.strings:5]: I have also noticed that you have a wrong date)

Comment (ngohungphuc) [reply Stats/Supporting Files/nl.lproj/Localizable.strings]: Resolved.

Comment (ngohungphuc) [reply Stats/Supporting Files/nl.lproj/Localizable.strings]: @exelban Please check again. Thanks you.

**Comment by exelban** [comment]
Oh, I forgot to ask. 
Can I add you as a contributor to this translation?

**Comment by ngohungphuc** [comment]
@exelban Done. Please check PR [404]( https://github.com/exelban/stats/pull/404/files)

**Pull request merged**